debug.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/*
+ * Plugin Name: Debug Loader
+ * Description: A set of debug tools and plugins
+ */
+
+
+/**
+ * Test Jquery Updates
+ *
+ * There are multiple versions per WordpressVersion the loader tries to pick the most recent one
+ */
+if ( ( defined( 'VIP_ENABLE_TEST_JQUERY_UPDATES' ) && VIP_ENABLE_TEST_JQUERY_UPDATES ) ) {
+	$base_folder = __DIR__ . '/debug/test-jquery-updates';
+	$entrypoint_script = 'wp-jquery-update-test.php';
+
+	$avaiable_options = array_diff( scandir( $base_folder ), [ '.', '..' ] );
+
+	$best_fit;
+	foreach ( $avaiable_options as $candidate ) {
+		if ( version_compare( $GLOBALS['wp_version'], $candidate, '>=' ) ) {
+			if ( ! $best_fit || version_compare( $candidate, $best_fit, '>' ) ) {
+				$best_fit = $candidate;
+			}
+		}
+	}
```

Comment:
What do you think about just having a hard-coded mapping and avoiding the added overhead from scandir + version_compare?

Reply:
I kinda like that this doesn't need any interaction if we would add a version like 5.5.3 >, and so on. Especially considering the test-jquery-plugins is not strictly versioned.

The downside being performance and a bit more complex code. For complex code, I think the variables naming is alright to make it easier to reason about.

Performance I think we only gonna use this in debug sessions where the performance is not the priority. Did I get that correctly?